EIPS/eip-4758.md

```diff
@@ -0,0 +1,39 @@
+---
+eip: 4758
+title: Deactivate SELFDESTRUCT
+description: Deactivate SELFDESTRUCT by changing it to SENDALL, which does recover all funds to the caller but does not delete any contract code or storage.
+author: Guillaume Ballet (@gballet), Vitalik Buterin (@vbuterin), Dankrad Feist (@dankrad)
+discussions-to: https://github.com/ethereum/EIPs/issues/4759
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-02-03
+requires: 
+---
+
+# Deactivating SELFDESTRUCT
+
+## Summary
+
+This EIP renames the `SELFDESTRUCT` opcode to `SENDALL`, and replaces its functionality. The new functionality will be only to send all Ether in the account to the caller.
+
+## Motivation
+
+The `SELFDESTRUCT` opcode requires large changes to the state of an account, in particular removing all code and storage. This will not be possible in the future with Verkle trees: Each account will be stored in many different account keys, which will not be obviously connected to the root account (see [Verkle tree EIP](https://notes.ethereum.org/5HDhQXstTaKtVqVbS7S9yw)).
```

Comment:
```suggestion
The `SELFDESTRUCT` opcode requires large changes to the state of an account, in particular removing all code and storage. This will not be possible in the future with Verkle trees: Each account will be stored in many different account keys, which will not be obviously connected to the root account.
```
No external links in EIPs, they should be self-contained and contain all necessary information inline.  Note: The motivation doesn't need to a be a fully convincing argument, just give people the gist of "why" and a starting point for them to research more if they want.  If you want to include links, I recommend adding them to the discussions-to link.